ballerina/Ballerina.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 org = "ballerina"
 name = "yaml"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
Don't we need to update the version as 0.7.0?

Reply:
Yes. changed with [de1d704](https://github.com/ballerina-platform/module-ballerina-yaml/pull/57/commits/de1d7048be915ad5d5d08729bdc794b370f06795)